utils/node_local_test.py

```diff
@@ -2385,77 +2420,204 @@ def test_cont_clone(self):
         destroy_container(self.conf, self.pool.id(), container)
 # pylint: enable=too-many-public-methods
 
+class nlt_stdout_wrapper():
+    """Class for capturing stdout from threads"""
+
+    def __init__(self):
+        self._stdout = sys.stdout
+        self._outputs = {}
+        sys.stdout = self
```

Comment:
It seems okay if it's working, but I wonder if there are any other side effects of overriding `sys.stdout = self`. E.g. if there is a missing function definition or attribute in `self`

Reply:
Seems to just be write() and flush().